support-frontend/assets/components/subscriptionCheckouts/summary.jsx

```diff
@@ -215,13 +217,17 @@ export default class Summary extends Component<PropTypes, StateTypes> {
   }
 
   render() {
+    const url = new URL(window.location).pathname;
```

Comment:
I think a better way to do this is to pass the `product` from the state into this component (you'll need to rebase onto master to get this field).

Reply:
Sure, shall do